main.py

```diff
@@ -109,52 +109,47 @@ def slideshow(db, start, end, seconds="0.25", rand=False):
     try:
         for x in r:
             pokemon = db.get_pokemon(x)
-            scripter.change_terminal(pokemon)
+            scripter.change_terminal(pokemon.get_path())
             time.sleep(delay)
     except KeyboardInterrupt:
         print("Program was terminated.")
         sys.exit()
 
 
-def change_terminal_background(db, arg):
-    # Change the terminal background to the specified Pokemon, if it exists.
+def change_terminal_background(db, arg):  # arg is a pokemon_name
+    """Change the terminal background to the specified Pokemon, if it exists."""
     if arg in db:
         pokemon = db.get_pokemon(arg)
-        scripter.change_terminal(pokemon)
+        scripter.change_terminal(pokemon.get_path())
     else:  # If not found in the database, try to give suggestions.
         suggestions = db.names_with_infix(arg)
-        if len(suggestions) == 0:
+        try:
+            pokemon = suggestions[0]
+            scripter.change_terminal(pokemon.get_path())
```

Comment:
I don't think we should shove a suggested pokemon down the user throat only if very obvious. I don't think the if here should be removed. We should only set the pokemon to the suggested one if there is only a single suggested one.